src/common/constant.ts

```diff
@@ -1 +1,7 @@
-export const USER_ACCOUNT_INFO_ENDPOINT = 'https://66a0bce27053166bcabc8e93.mockapi.io/api/v1/account/user';
+export const USER_ACCOUNT_INFO_ENDPOINT = '/api/v1/microservices';
+export const BASE_URL_SAAS = ' https://4auv2dy1jg.execute-api.ap-south-1.amazonaws.com'
+
+export function getSaveLogsEndpoint(organizationId: string, projectId: string) {
```

Comment:
why not use variable directly instead of function
import  required  variable here and export it